src/InputGroup.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components'
+import system, { space, borderColor, propTypes } from 'styled-system'
+import theme from './theme'
+import Box from './Box'
+import Input from './Input'
+
+const InputGroup = styled.div`
+  display: flex;
+  align-items: center;
+  border-radius: ${system.theme('radius')};
+  border-width: 1px;
+  border-style: solid;
+  ${borderColor}
+  ${space}
+
+  & > ${Box} {
+    width: 100%;
+    flex: 1 1 auto;
```

Comment:
@jxnblk ,
This `flex` seems to be a short-hand property for `flex-grow`, `flex-shrink`, and `flex-basis`. (https://developer.mozilla.org/en-US/docs/Web/CSS/flex)

Would you think it'd be too verbose if the code spells all off the implicit rules to avoid using short-hand property?

Also `flex` is only supported >= ie11. Would you have concern about ie10 support?
<img width="898" alt="flex-support" src="https://user-images.githubusercontent.com/6441326/35590820-f868b588-05d5-11e8-8d59-d21d0490e97c.png">

Reply:
I'll change to not use the shorthand. I *think* (but might be wrong) that the `width` property will help this work as expected in IE10

Reply:
I thought we had dropped support for <IE11.

Reply:
@chrisdroukas , think IE 10 may be still in "business". We can confirm with the "committee" that authors the browser support.